### PR TITLE
Add a repr to Entity to help with debugging

### LINKED ISSUE #211: EdEntity string representation
Print all relevant info instead of  '<robot_skills.util.entity.Entity at 0x7f9917b2ded0>'

### PR DESCRIPTION
**Pull request by LoyVanBeek**
Fixes #211

### CODE REVIEW

File: robot_skills/src/robot_skills/util/entity.py [hunk 1]
```diff
@@ -96,6 +96,13 @@ def pose(self, pose):
         """ Setter """
         self._pose = poseMsgToKdlFrame(pose)
 
+    def __repr__(self):
+        xyz = "(x={x}, y={y}, z={z})".format(x=self._pose.p.x(), y=self._pose.p.y(), z=self._pose.p.z())
+        r,p,y = self._pose.M.GetRPY()
```
Comment (jlunenburg) [inline robot_skills/src/robot_skills/util/entity.py]: pep compliancy

File: robot_skills/src/robot_skills/util/kdl_conversions.py [hunk 2]
```diff
@@ -18,7 +18,10 @@ def __init__(self, frame, frame_id, stamp=None):
         self.frame_id = frame_id
 
     def __repr__(self):
-        return "{frame} @ {fid}".format(frame=self.frame, fid=self.frame_id)
+        xyz = "(x={x}, y={y}, z={z})".format(x=self.frame.p.x(), y=self.frame.p.y(), z=self.frame.p.z())
+        r,p,y = self._pose.M.GetRPY()
```
Comment (jlunenburg) [inline robot_skills/src/robot_skills/util/kdl_conversions.py]: pep compliancy

File: robot_skills/src/robot_skills/util/entity.py [hunk 3]
```diff
@@ -96,6 +96,13 @@ def pose(self, pose):
...
+        rpy = "(r={x}, p={y}, y={z})".format(x=r, y=p, z=y)
+        frame = "{pos} x {rot}".format(pos=xyz, rot=rpy)
```
Comment (jlunenburg) [inline robot_skills/src/robot_skills/util/entity.py]: not a fan of the {pos} x {rot} --> this implies a multiplication

File: robot_skills/src/robot_skills/util/kdl_conversions.py [hunk 4]
```diff
@@ -18,7 +18,10 @@ def __init__(self, frame, frame_id, stamp=None):
...
+        rpy = "(r={x}, p={y}, y={z})".format(x=r, y=p, z=y)
```
Comment (jlunenburg) [inline robot_skills/src/robot_skills/util/kdl_conversions.py]: not a fan of the {pos} x {rot} --> this implies a multiplication

**Comment by MatthijsBurgh** [comment]
Lekker bezig @timassman

**Pull request merged**